composer.json

```
@@ -41,12 +41,19 @@
     "php-stubs/wp-cli-stubs": "^2.4",
     "phpunit/phpunit": "^6.5 || ^7.5",
     "szepeviktor/phpstan-wordpress": "^0.7.0",
-    "wearerequired/coding-standards": "^1.5",
+    "wearerequired/coding-standards": "^2.1.0",
```

```suggestion
    "wearerequired/coding-standards": "^1.5",
```

This should not be updated as part of this PR.